src/main/java/sirius/biz/jobs/batch/file/SimpleXMLValidator.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.jobs.batch.file;
+
+import org.xml.sax.ErrorHandler;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+import sirius.biz.process.ProcessContext;
+import sirius.biz.process.logs.ProcessLog;
+import sirius.kernel.commons.Monoflop;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.health.Exceptions;
+import sirius.web.security.UserContext;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.IOException;
+import java.util.Locale;
+
+/**
+ * wra
+ */
+public class SimpleXMLValidator {
```

Comment:
Why is this simple and what would be a complex one?

Reply:
I would say that it is simple as only one xml and xsd source can be provided without additional settings. But I can rename it to XMLValidator as this class might evolve in time.

Reply:
XsdParser or XsdValidator would be my call

Reply:
In my opinion, XsdValidator makes it seem like it validates XSD's.